src/components/TextField/TextField.tsx

```diff
@@ -24,12 +25,17 @@ export const TextField: React.FC<Props> = ({
   required = false,
   onChange = () => {},
 }) => {
-  // generage a unique id once on component load
   const [id] = useState(() => `${name}-${getRandomDigits()}`);
-
-  // To show errors only if the field was touched (onBlur)
+  const thisIsUrl = name === 'imdbUrl' || name === 'imgUrl';
```

Comment:
'this' is a service word, like a var or function.
so please don't use service words for the variable name.
```suggestion
  const isUrl = name === 'imdbUrl' || name === 'imgUrl';
```

Reply:
done